kyo-prelude/jvm/src/test/scala/kyo/kernel/BytecodeTest.scala

```diff
@@ -32,12 +32,12 @@ class BytecodeTest extends Test:
 
     "map" in {
         val map = methodBytecodeSize[TestMap]
-        assert(map == Map("test" -> 18, "anonfun" -> 10, "mapLoop" -> 154))
+        assert(map == Map("test" -> 18, "anonfun" -> 10, "mapLoop" -> 132))
```

Comment:
Reduction in bytecode size because `Safepoint.enter` now returns a `Boolean` instead of `Long` and the callers don't need to wire the `depth` to `Safepoint.exit`.